src/web/components/Tabs/Tabs.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { TabsBase } from './';
+import List from '../List';
+import Touchable from '../Touchable';
+
+describe('Tabs', () => {
+  let wrapper;
+  let onSelected;
+  let data;
+
+  beforeEach(() => {
+    data = ['Hello', 'Bonjour', 'Ciao'];
```

Comment:
What do you think of something a bit powerful ?

```javascript
const tabs = [{
 key: 'Hello',
 component: () => <SomeComponent />
}]
```

Reply:
So with the internal state management, `Tabs` is independant but respect its own responsability: display a content tab when its focused